### DISCUSSION TITLE
Why does selecting a folder for copying copy the entire folder instead of just the differences?

### DISCUSSION THREAD

**Discussion by chinagreenelvis** [WinMerge/winmerge#1773] [Q&A]
If I don't use tree view mode, Winmerge becomes unresponsive. If I use tree view mode, I have to actually open up each individual folder to merge the differences by selecting only the files. Otherwise, it copies the entire contents of the folder.

Is there some way of not doing this?

**Comment by sdottaka** [comment]
Unfortunately, the only way to copy only the different files is to select the different files rather than the folder.

There is also the following issue, and I think that something needs to be improved, but nothing has been implemented yet.

https://github.com/WinMerge/winmerge/issues/198

**Comment by chinagreenelvis** [comment]
This definitely means I have outgrown WinMerge as a solution for synchronizing my files with backup drives. Beyond Compare is able to do this at the folder level, although it seems like the file operations may be a little on the slow side.
